src/lib/Enum/actions.ts

```diff
@@ -0,0 +1,3 @@
+export const ACTION_DOWNLOAD = 'Télécharger le tableau';
+export const ACTION_ENROLE = 'Enrôler un patient';
+export const ACTION_IMPORT = 'Importer des fichiers';
```

Comment:
```suggestion
export enum Actions = {
  Download = 'Télécharger le tableau',
  ...
}
```

Renommer le dossier `Enum` en `enums`